tests/console/machinery.pm

```diff
@@ -13,13 +13,37 @@
 use base "consoletest";
 use strict;
 use testapi;
+use utils;
+use version_utils;
 
 sub run {
     select_console 'root-console';
-    assert_script_run 'which OneClickInstallCLI || zypper -n in yast2-metapackage-handler', 200;
-    my $ret_check = '[ "$?" = "141" ] || [ "$?" = "0" ]';
-    assert_script_run 'yes | OneClickInstallCLI http://machinery-project.org/machinery.ymp ; ' . $ret_check, 200;
+
+    if (is_sle) {
+        assert_script_run 'source /etc/os-release';
+        if (is_sle '>=15') {
```

Comment:
Is this also true for SLE15 or only SLE15SP1?

Reply:
@okurz I could change it to SLE15SP1 if you like.